problems/2.functions.js

```diff
@@ -2,36 +2,43 @@
 * @param {string} name - The name of the user to greet
 * @returns {null}
 */
-function logGreeting() {
-
+function logGreeting(name) {
+  console.log("Hello " + name + '!')
 }
+logGreeting("Tom ");  
 
 /** Returns a custom greeting string in the form "Hello Avery!"
 * @param {string} name - The name of the user to greet
 * @returns {string} - The custom greeting
 */
-function greeting() {
-
+function greeting(name) {
+  return "Hello " + name + "!";
 }
+console.log(greeting("Corey "))
 
 /** Adds a number to a global variable called globalCount
  * Access globalCount like this: global.globalCount
 * @param  {number} val - The number to add to globalCount
 * @returns {null}
 * @modifies {global.globalCount}
 */
-function incrementGlobalCount() {
-
+function incrementGlobalCount(val) {
+global.GlobalCount = 0
+return val + global.GlobalCount
 }
```

Comment:
To update a variable you need to do reassignment and thus need the assignment operator (=). 
This problem is saying global.globalCount already exists, so don't set it  0.
global.globalCount += val